connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -250,8 +250,9 @@ object KustoWriter {
 
   def createBlobWriter(tableCoordinates: KustoCoordinates,
                        tmpTableName: String,
-                       client: ExtendedKustoClient): BlobWriteResource = {
-    val blobName = s"${KustoQueryUtils.simplifyName(tableCoordinates.database)}_${tmpTableName}_${UUID.randomUUID.toString}_spark.csv.gz"
+                       client: ExtendedKustoClient,
+                       partitionId: String): BlobWriteResource = {
+    val blobName = s"${KustoQueryUtils.simplifyName(tableCoordinates.database)}_${tmpTableName}_${UUID.randomUUID.toString}_${partitionId}_spark.csv.gz"
 
     val containerAndSas = client.getTempBlobForIngestion
     val currentBlob = new CloudBlockBlob(new URI(containerAndSas.containerUrl + '/' + blobName + containerAndSas.sas))
```

Comment:
Small nit
new URI(s"${containerAndSas.containerUrl}/$blobName${containerAndSas.sas}) just like the blobName field